src/modules/UI/components/QRCode/QRCode.ui.js

```diff
@@ -1,20 +1,26 @@
+// @flow
```

Comment:
This PR is for the FlipInput. What changes were needed in this file for the FlipInput animation?